src/main/java/edu/kis/vh/nursery/ListIntegers/IntArrayStack.java

```diff
@@ -0,0 +1,52 @@
+package edu.kis.vh.nursery.ListIntegers;
+
+import edu.kis.vh.nursery.ListInterface;
+
+public class IntArrayStack implements ListInterface {
+
+    public int MAX_AMOUNT_OF_ELEMENTS = 12;
+    final public int[] numbers = new int[MAX_AMOUNT_OF_ELEMENTS];
+
+    private int EMPTY = -1;
+    private int VALUE_IF_LIST_EMPTY = 0;
+
+
+    @Override
+    public void push(int in) {
+        if (!isFull())
+            numbers[++EMPTY] = in;
+    }
+
+
+    @Override
+    public boolean isEmpty() {
+        return EMPTY == -1;
+    }
+
+
+    public boolean isFull() {
+        return EMPTY == MAX_AMOUNT_OF_ELEMENTS - 1;
+    }
+
+
+    @Override
+    public int top() {
+        if (isEmpty())
+            return VALUE_IF_LIST_EMPTY;
```

Comment:
Use it to do tasks:
5. Czy masz stałe wspólne dla IntArrayStack i IntLinkedList? Jeżeli nie przeanalizuj odpowiedzialności poszczególnych typów w tym zakresie i w razie potrzeby popraw kod. (Refactor → Move lub alt+shift+v).
6. Zmień wartość zwracaną przez metody peekaboo i countOut w przypadku pustego stosu z -1 na 0.